software/cloud-dashboard/react-client/src/components/plot.js

```diff
@@ -92,7 +92,7 @@ class Plot extends Component {
             <ResponsiveContainer width="94%" height={300}>
                 <LineChart data={dataToShow}>
                     <XAxis dataKey="time" label="Date" interval={tickInterval} tickFormatter={this.formatDate} />
-                    <YAxis />
+                    <YAxis type="number" domain={[this.props.yAxisMinMax[0], this.props.yAxisMinMax[1]]}/>
```

Comment:
`props.yAxisMinMax` is not available here yet, it needs to be passed into this `Plot` component from the `PlotCard` component and you should be good to use it!

Reply:
Fixed